### PR TITLE
sentry: adds additionally integrations

### PR DESCRIPTION
**Pull request by rerowep**

### CODE REVIEW

File: invenio_logging/config.py [hunk 1]
```diff
@@ -97,8 +97,13 @@
 LOGGING_SENTRY_PYWARNINGS = False
 """Enable logging of Python warnings to Sentry."""
 
-LOGGING_SENTRY_CELERY = False
-"""Configure Celery to send logging to Sentry."""
+LOGGING_SENTRY_INTEGRATIONS = []
+"""Addidionaly Senty integrations for logging."""
```
Comment (greut) [inline invenio_logging/config.py:101]:
```suggestion
"""Additionally Sentry integrations for logging."""
```

Comment (rerowep) [reply invenio_logging/config.py:101]: ok

**Comment by jma** [comment]
Any news? I would be very usefull on production. Thanks.

**Comment by lnielsen** [comment]
Closed by #64

**Pull request closed without merging**